docker-compose.yml

```
@@ -0,0 +1,35 @@
+version: '3'
+services:
+  admin:
+    user: "www-data:www-data"
+    build: ./admin_panel
+    env_file:
+      - ./.env
```

см [это](https://github.com/mikhail349/notifications_sprint_1/pull/11/files#r1097105286)
```suggestion
      - ./admin_panel/.env
```